src/Common/src/Interop/Windows/mincore/Interop.GetFullPathNameW.cs

```diff
@@ -22,19 +22,25 @@ partial class mincore
         [DllImport(Libraries.CoreFile_L1, EntryPoint = "GetFullPathNameW", SetLastError = true, CharSet = CharSet.Unicode, BestFitMapping = false)]
         private static extern int GetFullPathNamePrivate(string path, int numBufferChars, [Out]StringBuilder buffer, IntPtr mustBeZero);
 
-        internal static int GetFullPathName(string path, int numBufferChars, [Out]StringBuilder buffer, IntPtr mustBeZero)
+        internal static int GetFullPathName(string path, int numBufferChars, [Out]StringBuilder buffer)
```

Comment:
As an aside, I'm starting to worry about the amount of logic that's crept into the interop files.  Are there any projects that depend on these interop definitions and that don't also need PathInternal, or that couldn't be made to easily use PathInternal?  I know we previously had a discussion about this, but I'm thinking now we should move this logic all out of the interop files into PathInternal, which the projects that need the logic can use.  That then also helps avoid what feels like a backwards dependency: the interop files using PathInternal.